README.md

```diff
@@ -54,11 +77,16 @@ In this phase everyone is thinking like a Designer.  Design thinking also relies
 
 ***Q1.5 Based on the feedback, how might you change your solution?***
 
+* coffee machine knows when alarm is and starts brewing so it is hot and ready right before the alarm goes off
```

Comment:
Having the alarm and the coffee machine in communication with one another is definitely feasible!